cypress/integration/login.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Login Page
```

Comment:
can we have feature files installed in a separate or sub folder than the step definitions for proper project structuring

Reply:
So, on the organisation structure, I'm generally following the guidance [from the preprocessor plugin](https://github.com/TheBrainFamily/cypress-cucumber-preprocessor#step-definitions), which allows us to have reusable step definitions where it makes sense but also not to pollute a global namespace where it doesn't make sense. So, yes, this setup will allow us to have step definitions that are separate from a feature file.

In this case, however, pretty much none of the Given, When, or Then steps (except the one for `Then(/the error message "(.+)" should be visible/)`) are generic. E.g., I would strongly **not** recommend anyone trying to reuse these step definitions as a way to login to the app. For that purpose, there's a [`cy.login()` command](https://github.com/openmrs/openmrs-ocl-client/blob/823d76cdaee8e14708251ca9b115ebf13dc9beae/cypress/support/commands.ts#L11) that provides the necessary functionality. And this is, generally speaking, the pattern I think we should follow: parts concerned with ensuring the application state is in a certain configuration should be written as Cypress commands so that the step definitions are nothing more than bundles of Cypress commands to perform and verify actions.

This also allows us to isolate features, so, e.g., if the login page doesn't work, we don't break every other test, because only the login page feature interacts with the actual login page.